IRC.md

```diff
@@ -1,111 +1,238 @@
 # Connecting to IRC
 
-Twitch offers an IRC interface to its chat. This allows for people to do things like develop bots for their channel, or to connect to a channel's chat with an IRC client instead of using the web interface. While our IRC server is built to follow [RFC1459](http://tools.ietf.org/html/rfc1459.html), it is important to note that there are some cases where it will behave slightly differently than another IRC server would. These cases are noted when necessary in the following document.
+Twitch offers an IRC interface to its chat. This allows for people to do things like develop bots for their channel, or to connect to a channel's chat with an IRC client instead of using the web interface. While our IRC server is built to follow [RFC1459](http://tools.ietf.org/html/rfc1459.html), it is important to note that there are **several** cases where it will behave slightly differently than another IRC server would. These cases are noted when necessary in the following document.
```

Comment:
Please do not use bold except for when absolutely necessary, and if used it should be over a full thought, not a single word.